tensorflow-sources.spec

```diff
@@ -0,0 +1,54 @@
+### RPM external tensorflow-sources 1.1.0
+Source: https://github.com/tensorflow/tensorflow/archive/v%{realversion}.tar.gz
+
+BuildRequires: bazel
+Requires: py2-numpy python py2-wheel
+
+%prep
+
+%setup -q -n tensorflow-%{realversion}
+
+%build
+export PYTHON_BIN_PATH=`which python`
+export TF_NEED_JEMALLOC=0
+export TF_NEED_HDFS=0
+export CC_OPT_FLAGS="-march=native"
```

Comment:
I guess this is connected to the "illegal instruction" crash I get in https://github.com/cms-sw/cmssw/pull/19893#issuecomment-324371051